doc/man/8/ceph-create-keys.rst

```diff
@@ -0,0 +1,63 @@
+:orphan:
+
+===============================================
+ceph-create-keys -- ceph keyring generate tool
+===============================================
+
+.. program:: ceph-create-keys
+
+Synopsis
+========
+
+| **ceph-create-keys** [-h] [-v] [--cluster *name*] --id *id*
+
+
+Description
+===========
+
+:program:`ceph-create-keys` is a utility to generate bootstrap keyrings using
+the given monitor when it is ready.
+
+it creates following auth entities (or users)
```

Comment:
Sorry didn't notice that this was a backport PR.. Guess it should be fine either way